Add relevant docs to enable HashiCorp AppRole method (#4667)
wso2/docs-apim: en/docs/install-and-setup/setup/mi-setup/security/using-hashicorp-secrets.md

    </tr>
</table>

!!! Note

>     Refer [ACL rules](https://www.vaultproject.io/docs/secrets/kv/kv-v2#acl-rules) and [configuration](https://www.vaultproject.io/docs/auth/approle#configuration) sections to enable AppRole auth method in HashiCorp secure vault.

Reviewer: ```suggestion
    Refer [ACL rules](https://www.vaultproject.io/docs/secrets/kv/kv-v2#acl-rules) and [configuration](https://www.vaultproject.io/docs/auth/approle#configuration) sections to enable the AppRole authentication method in HashiCorp secure vault.
```

>     Refer [ACL rules](https://www.vaultproject.io/docs/secrets/kv/kv-v2#acl-rules) and [configuration](https://www.vaultproject.io/docs/auth/approle#configuration) sections to enable AppRole auth method in HashiCorp secure vault.

Reviewer 2: ```suggestion
    For information on enabling AppRole auth method in HashiCorp secure vault, see the [ACL rules](https://www.vaultproject.io/docs/secrets/kv/kv-v2#acl-rules) and [configuration](https://www.vaultproject.io/docs/auth/approle#configuration) sections.
```
